src/java/org/apache/cassandra/cql3/Operator.java

```diff
@@ -258,6 +258,20 @@ public boolean isSatisfiedBy(AbstractType<?> type, ByteBuffer leftOperand, ByteB
         {
             throw new UnsupportedOperationException();
         }
+    },
+    ANN(15)
+    {
+        @Override
+        public String toString()
+        {
+            return "ANN";
+        }
+
+        @Override
+        public boolean isSatisfiedBy(AbstractType<?> type, ByteBuffer leftOperand, ByteBuffer rightOperand)
+        {
+            return true;
```

Comment:
This probably deserves a comment explaining why it is no-op.

Reply:
I have changed the implementation to throw a UOE and documented the reason. This method should never be called because the ANN operator is only supported by the index. This in the same as the LIKE operator.

Reply:
Note: If have reverted the change and put an explanation of why it needs to return true here.